src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js

```diff
@@ -560,17 +731,23 @@ function GraphsVisualizationsCtrl($scope, $rootScope, $repositories, $licenseSer
                     fixed: node.fixed
                 };
             });
+
+            const triplesCopy = JSON.stringify(Array.from(this.triples.entries()));;
+
             const linksCopy = _.map(this.links, function (link) {
                 return {
                     source: link.source.iri,
+                    tripleSource: link.source.isTriple,
```

Comment:
If this is boolean, maybe isTripleSource/isTripleTarget is more appropriate for the key